### DISCUSSION TITLE
@nuxtjs/storybook help needed

### DISCUSSION THREAD

**Discussion by Rigo-m** [storybookjs/storybook#29688] [Help] [answered]
### Summary

Dear storybook team, a couple of days ago this issue popped up https://github.com/nuxt-modules/storybook/issues/813 in the nuxt storybook repository. Currently, storybook+chromatic are the state of the art design-system+visual regression tests suite and we are super happily using this combo in all of our nuxt projects. Is there any way that the storybook team (@JReinhold ) can help @tobiasdiez pushing the envelope and fix the upstream issues blocking us nuxt users from enjoying latest storybook versions?

P.s.: props to the storybook team for being such a prime example of what the e18e project can achieve!

**Comment by shilman** [comment, marked as answer]
Thanks so much for bringing this to our attention. Responded on the other issue! 🙏
